bin/run-gentleman-jerry.sh

```diff
@@ -20,57 +20,19 @@ if [[ -n "$SSL_KEY" ]]; then
   mkdir -p "$(dirname "$SSL_KEY_FILE")"
   touch "$SSL_KEY_FILE"
   chmod 600 "$SSL_KEY_FILE"
-  echo "$SSL_KEY" > "$SSL_KEY_FILE"
+  echo "$SSL_KEY" >"$SSL_KEY_FILE"
```

Comment:
Unimportant, but why delete the space?

Reply:
oh, this was unintentional, probably just a stray backspace